src/components/end-test-link/end-test-link.ts

```diff
@@ -17,7 +17,7 @@ export class EndTestLinkComponent {
     this.terminateTestModal = this.modalController.create('TerminateTestModal', {
       onCancel: this.onCancel,
       onTerminate: this.onTerminate,
-    });
+    }, { cssClass: 'mes-modal-alert text-zoom-regular' });
```

Comment:
what's this for?

Reply:
`mes-modal-alert` allows the CSS rules to target modals that are to be opened as alerts (window centred on screen) and not other modals like Candidate Details which are full screen. 

`text-zoom-regular` is really a stop-gap until we address accessibility on pages other than the journal page. Chatting to Szabi, a provider to return the zoom level is probably needed.

Reply:
![Screenshot 2019-05-02 at 17 49 18](https://user-images.githubusercontent.com/8069071/57092159-a2c9b100-6d02-11e9-89d4-88c0e5f21d9f.png)

`ion-modal` is higher in the DOM than `ion-nav` so it doesn't have the zoom class that's used to set font size & line height for h2, h3 etc.